graylog2-web-interface/webpack.config.js

```diff
@@ -96,7 +88,17 @@ const webpackConfig = {
   },
   module: {
     rules: [
-      { test: /\.[jt]s(x)?$/, use: BABELLOADER, exclude: /node_modules|\.node_cache/ },
+      {
+        test: /\.[jt]s(x)?$/,
+        use: {
+          loader: 'babel-loader',
+          options: {
+            cacheDirectory: 'target/web/cache',
+            presets: ['graylog'],
```

Comment:
This is required although the preset is loaded in [`babel.config.js`](https://github.com/Graylog2/graylog2-server/pull/12713/files#diff-5e11c36d4d0224d96733c30b9a880bc5fff2ce82df160df7e1f8d5ce70a8d661), because `babel-loader` is not reading `babel.config.js` by default, so we duplicate it here.